Infusion.LegacyApi/Legacy.cs

```diff
@@ -444,6 +444,14 @@ public void TargetTile(string tileInfo)
             Targeting.TargetTile(tileInfo);
         }
 
+        public void TargetTile(int x, int y)
+        {
+            CheckCancellation();
+
+            NotifyAction();
+            Targeting.TargetTile(new Location2D(x, y));
```

Comment:
This is a better place for `var tiles = Ultima.Map.Felucca.Tiles.GetStaticTiles(location.X, location.Y);`.